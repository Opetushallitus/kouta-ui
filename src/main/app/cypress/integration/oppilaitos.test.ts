import { createOppilaitosForm } from './oppilaitosForm/createOppilaitosForm';
import { editOppilaitosForm } from './oppilaitosForm/editOppilaitosForm';

describe('createOppilaitos', createOppilaitosForm);
describe('editOppilaitos', editOppilaitosForm);
