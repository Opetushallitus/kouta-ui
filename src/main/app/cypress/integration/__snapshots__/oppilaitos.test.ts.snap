exports[`editOppilaitos > should be able to edit oppilaitos #0`] =
{
  "esikatselu": false,
  "kielivalinta": [
    "fi"
  ],
  "metadata": {
    "akatemioita": 1,
    "esittely": {
      "fi": "<p><em>Fi esittely</em></p>"
    },
    "hakijapalveluidenYhteystiedot": null,
    "kampuksia": 3,
    "korkeakouluja": 5,
    "opiskelijoita": 100,
    "tiedekuntia": 4,
    "tietoaOpiskelusta": [
      {
        "otsikkoKoodiUri": "organisaationkuvaustiedot_0#1",
        "teksti": {
          "fi": "<p>Fi tiedot</p>"
        }
      }
    ],
    "toimipisteita": 9,
    "yhteystiedot": [
      {
        "nimi": {
          "fi": "Yhteystiedon nimi"
        },
        "postiosoite": {
          "osoite": {
            "fi": "Fi osoite"
          },
          "postinumeroKoodiUri": "posti_0#2"
        },
        "kayntiosoite": null,
        "puhelinnumero": {
          "fi": "1234"
        },
        "sahkoposti": {
          "fi": "fi@sahkoposti.fi"
        }
      }
    ],
    "yksikoita": 2,
    "wwwSivu": {
      url: {
        "fi": "http://www.verkkosivu.fi"
      },
      nimi: {
        "fi": "Verkkosivu fi"
      }
    }
  },
  "muokkaaja": "1.2.246.562.24.62301161440",
  "oid": "1.1.1.1.1.1",
  "organisaatioOid": "1.1.1.1.1.1",
  "tila": "tallennettu"
};

exports[`createOppilaitos > should be able to create oppilaitos without hakijapalveluyhteystiedot #0`] =
{
  "esikatselu": false,
  "kielivalinta": [
    "fi"
  ],
  "metadata": {
    "akatemioita": 7,
    "esittely": {
      "fi": "<p>Esittely</p>"
    },
    "hakijapalveluidenYhteystiedot": null,
    "kampuksia": 4,
    "korkeakouluja": 2,
    "opiskelijoita": 1,
    "tiedekuntia": 3,
    "tietoaOpiskelusta": [
      {
        "otsikkoKoodiUri": "organisaationkuvaustiedot_02#1",
        "teksti": {
          "fi": "<p>Tietoa</p>"
        }
      }
    ],
    "toimipisteita": 6,
    "yhteystiedot": [
      {
        "nimi": {
          "fi": "Yhteystiedon nimi"
        },
        "postiosoite": {
          "osoite": {
            "fi": "Osoite"
          },
          "postinumeroKoodiUri": "posti_00350#2"
        },
        "kayntiosoite": {
          "osoite": {
            "fi": "Osoite"
          },
          "postinumeroKoodiUri": "posti_00350#2"
        },
        "puhelinnumero": {
          "fi": "12345"
        },
        "sahkoposti": {
          "fi": "sahkoposti@sahkoposti.fi"
        },
      }
    ],
    "yksikoita": 5,
    "wwwSivu": {
      url: {
        "fi": "http://www.verkkosivu.fi"
      },
      nimi: {
        "fi": "Verkkosivu fi"
      }
    }
  },
  "muokkaaja": "1.2.246.562.24.62301161440",
  "oid": "1.1.1.1.1.1",
  "organisaatioOid": "1.1.1.1.1.1",
  "tila": "julkaistu"
};

exports[`createOppilaitos > should be able to create oppilaitos with hakijapalveluyhteystiedot #0`] =
{
  "esikatselu": false,
  "kielivalinta": [
    "fi"
  ],
  "metadata": {
    "akatemioita": null,
    "esittely": {},
    "kampuksia": null,
    "korkeakouluja": null,
    "opiskelijoita": null,
    "tiedekuntia": null,
    "tietoaOpiskelusta": [],
    "toimipisteita": null,
    "yksikoita": null,
    "wwwSivu": null,
    "yhteystiedot": [
      {
        "nimi": {
          "fi": "Yhteystiedon nimi"
        },
        "postiosoite": {
          "osoite": {
            "fi": "Osoite"
          },
          "postinumeroKoodiUri": "posti_00350#2"
        },
        "kayntiosoite": {
          "osoite": {
            "fi": "Osoite"
          },
          "postinumeroKoodiUri": "posti_00350#2"
        },
        "puhelinnumero": {
          "fi": "12345"
        },
        "sahkoposti": {
          "fi": "sahkoposti@sahkoposti.fi"
        }
      }
    ],
    "hakijapalveluidenYhteystiedot": {
      "nimi": {
        "fi": "Testihakijapalvelu"
      },
      "postiosoite": {
        "osoite": {
          "fi": "Osoite"
        },
        "postinumeroKoodiUri": "posti_00350#2"
      },
      "kayntiosoite": {
        "osoite": {
          "fi": "Osoite"
        },
        "postinumeroKoodiUri": "posti_00350#2"
      },
      "puhelinnumero": {
        "fi": "12345"
      },
      "sahkoposti": {
        "fi": "sahkoposti@sahkoposti.fi"
      }
    }
  },
  "muokkaaja": "1.2.246.562.24.62301161440",
  "oid": "1.1.1.1.1.1",
  "organisaatioOid": "1.1.1.1.1.1"
};
